CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Added `crossOrigin` prop to `Tumbnail` component
 -   Added `isFollowingWindowSize` prop to `Tumbnail` component
 -   Added `resizeDebounceTime` prop to `Tumbnail` component
+-   Added `contentRef` prop to `Dialog` component
```

Comment:
It must be added under Unreleased > Added

Reply:
Sorry ! Done